### DISCUSSION TITLE
How does Webstudio compare to Bootstrap Studio?

### DISCUSSION THREAD

**Discussion by thegrasshopper104** [webstudio-is/webstudio-community#61] [General]
Hi,
First of all, congrats on the release and good luck with your business model!

I'm currently using [Bootstrap Studio](https://bootstrapstudio.io/), and I'm pretty happy with it. It abstracts a lot of things but still lets me see what's the actual CSS. Of course CSS is still a pain, but as far as WYSIWYG goes it's a superb app.

So, I'm wondering, what can Webstudio do that BSS can't or do it better?

I've played around with it a bit and so far didn't notice a big difference in the UX

**Comment by kof** [comment]
You answered it yourself haha: "CSS is still a pain". Also we are building a next level UX.

Beyond that I recommend checking out: 
landing: https://webstudio.is
blog: https://webstudio.is/blog
youtube: https://www.youtube.com/@getwebstudio

**Comment by kof** [comment]
Mainly webstuido is not just a static site generator. You will be able to build applications in the future.

**Reply by thegrasshopper104** [reply]
Thanks. I can definitely see myself switching if it'll help keep content and frontend in one place in the future.

**Reply by kof** [reply]
If you are using it as a html/css generator to paste it somewhere - then we don't have it yet. We are generating a Remix app, not a static html/css atm. 

Static may come some day, but not now.

**Reply by thegrasshopper104** [reply]
Of course having static exports is important, but if you'll make something where content management can be integrated inside the platform, it's going to be much less important.
